src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd

```diff
@@ -1,20 +1,21 @@
 @echo off
 setlocal
 
```

Comment:
I think you need elevated permissions to do all this, right? If so, consider detecting if the script is running elevated and bailing out if not with Usage. Suggestion only